### DISCUSSION TITLE
Cannot drag and place models on wall in iOS

### DISCUSSION THREAD

**Discussion by streetcodec** [google/model-viewer#4387] [Q&A] [answered]
I have used ar-placement = "wall" attribute and it is working fine in Android, but in iOS Device, it requires floor in view and then when the model is placed, I am unable to drag it upwards, in y axis. Did anyone else encounter this and found any workarounds?

**Comment by danilokrulj** [comment, marked as answer]
You can drag the model on the y axis using two finger drag (up/down). Not sure if anything is wrong with wall placement though.
